### DISCUSSION TITLE
Server naming the bot something homophobic because of it's pfp

### DISCUSSION THREAD

**Discussion by LakeyTheSeamonster** [machineonamission/mediaforge#108] [General]
yeah this server i'm in made the mediaforge's nickname the f slur, i thought i would bring this to your attention <3

![image_2022-06-25_211810881](https://user-images.githubusercontent.com/108207075/175799183-c5dd49eb-a3bf-4a02-a576-91ab7cc75fef.png)

the server id is 922960751184969808

**Comment by machineonamission** [comment]
thanks for telling me, i'll quickly add a thing so I can ban servers from the bot.

**Comment by machineonamission** [comment]
![image](https://user-images.githubusercontent.com/35177464/175801139-b396484f-f06d-4dfe-8852-67a7cafb2f50.png)
